src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -298,9 +332,22 @@ const getProps = (props: Props) => {
   }
 }
 
-const renderPricing = (salePrice, saleDate, user, mediator, size) => {
+const renderPricing = (
+  salePrice,
+  saleDate,
+  user,
+  mediator,
+  size,
+  filtersHaveUpdated,
+  paginated
+) => {
   const textSize = size === "xs" ? "2" : "3t"
-  if (user) {
+
+  // If user is logged in we show prices. Otherwise we show prices only for the default view - on page 1 and filters not changed.
+  // Ideally we get current page number from filter context 'page' property but somehow it is always '1'.
+  // So we resort to pagination detection. If user has paginated at all, prices will be hidden. even if user comes back to page 1.
+  // TODO: Fix filter context so its 'page' property has the current page number, then change this code.
```

Comment:
Interesting discovery around the filter context page bug! Nice find!

I'm not super familiar with filter contexts but I do think we have another one working on the Artwork page. 
https://github.com/artsy/reaction/blob/master/src/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx#L26

This one also handles some logic around tracking the `page` property. Would be curious to see how that one handles this and if we can use that to help us debug the page number bug you found.

Reply:
Oh. I will check that out. Thanks again!